ofonotest/qml/ofonotest/main.qml

```diff
@@ -1,74 +1,148 @@
 import QtQuick 2.0
 import QOfono 0.2
 
-
 Rectangle {
     width: 480
     height: 560
-    Text {
-        id: textLine
-        anchors.centerIn: parent
-    }
-    Text {
-        id: textLine2
-        anchors.top: textLine.bottom
-        text: manager.available ? netreg.name : "Ofono not available"
+
+    Column {
+
+        anchors.fill: parent
+
+        Text {
+            text: context1.active ? "online" : "offline"
+        }
+
+//        Text {
+//            text:  "modem Manufacturer " + modem1.manufacturer
+//        }
+
+        Text {
+            id: textLine2
+
+            text: manager.available ? netreg.name : "Ofono not available"
+        }
+
+        Text {
+            text: "simManager.pinRequired " + simManager.pinRequired
+            visible: simManager.pinRequired !== OfonoSimManager.NoPin
+        }
+
+        Repeater {
+            model: manager.modems
+
+            Text {
+                text: modelData
+                font.bold: modelData === manager.defaultModem
+            }
+
+        }
+
     }
+
     MouseArea {
         anchors.fill: parent
         onClicked: {
-          context1.active = !context1.active
+            context1.active = !context1.active;
```

Comment:
We don't use semicolons at the end of the lines.

Reply:
I will remove that. The semicolon was probably added by qmlformat.